addons/project/tests/test_task_dependencies.py

```diff
@@ -147,6 +147,13 @@ def test_duplicate_project_with_task_dependencies(self):
         self.assertEqual(task1_copy_copy.depend_on_ids.ids, [self.task_1.id],
                          "Copy should not alter the relation if the other task is in a different project")
 
+        self.project_pigs.allow_task_dependencies = False
+        project_pigs_no_dep = self.project_pigs.copy()
+        self.assertFalse(project_pigs_no_dep.allow_task_dependencies, 'The copied project should have the dependencies feature disabled')
+        for task in project_pigs_no_dep.task_ids:
+            self.assertFalse(task.depend_on_ids, 'The copied task should not have any dependencies')
+            self.assertFalse(task.dependent_ids, 'The copied task should not have any dependencies')
```

Comment:
Equivalent to that if you want. 🙂 

```suggestion
        self.assertFalse(project_pigs_no_dep.task_ids.depend_on_ids, 'The copied task should not have any dependencies')
        self.assertFalse(project_pigs_no_dep.task_ids.dependent_ids, 'The copied task should not have any dependencies')
```